ebu_tt_live/node/consumer.py

```diff
@@ -46,3 +45,23 @@ def reference_clock(self):
     @reference_clock.setter
     def reference_clock(self, value):
         self._reference_clock = value
+
+
+class EBUTTDConverterConsumer(SimpleConsumer):
```

Comment:
This can be done later, but I would change the name to "Encoder" to be consistent with the spec.